idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/lineMarker/LineMarkerSyntax.kt

```diff
@@ -115,4 +199,38 @@ interface LineMarkerSyntax {
           override fun getClickAction(): AnAction? = clickAction
         }
     }
+
+  fun <R> LineMarkerSyntax.navigationGutter(
+    icon: Icon,
+    element: PsiElement,
+    config: NavigationGutterIconBuilder<PsiElement>.(element: PsiElement) -> R
+  ): R =
+    NavigationGutterIconBuilder.create(icon).config(element)
```

Comment:
this seems not in use in the code and might not be needed, as the replacement is almost identical. What's the use-case?

Reply:
I'll add Docs to it.